spec/controllers/tasks_controller_spec.rb

```diff
@@ -28,5 +28,28 @@
             get :index
             expect(JSON.parse(response.body).size).to eq(2)
         end
-    end 
-end
\ No newline at end of file
+    end
+    
+    describe "DELETE#destroy" do
+
+        it "returns http success" do
+            task = Task.create(description: "walk the dog")
+            delete :destroy, params: {id: task.id}
+            expect(response).to have_http_status(:success)
+        end
+
+        it "returns a message that the task was deleted" do
+            task1 = Task.create(description: "walk the dog")
+            delete :destroy, params: {id: task1.id}
+            expect(response.body).to eq("The task was successfully deleted.")
+        end
+
+        it "will delete a task" do
+            task1 = Task.create(description: "walk the dog")
+            expect{
+                delete :destroy, params: {id: task1.id}
+            }.to change(Task, :count).by(-1)
+        end
```

Comment:
np: typically you want a line between your test setup, assertion, teardown (implicit in Ruby). So for example you could add a new line between lines 37 (test setup), and line 38 (assertion). It helps for readability so you can look at it and know "ok this is what it takes to setup this test, then a line break, and this is what I am expecting out of this test."

Reply:
Fix that, looks nicer!